contributors/devel/contributing-api-changes.md

```diff
@@ -0,0 +1,154 @@
+considered accepted once the proposal PR has been merged.
+
+*Feature process*
+
+All API changes require an issue to be filed in the [kubernetes/features] repo.  The
+issue will be used by release coordinators to track which features are slated for a
+release milestone.
+
+## Changes that qualify as API changes
+
+All changes to existing or new APIs are subject to the process outlined in this document.
+
+The following qualify as API changes:
+
+- Adding, removing, or modifying API types (e.g. through types.go, or by defining a new annotation)
+- Changing the version of an existing type (e.g. v1alpha1 to v1beta1)
+- Adding, removing, or modifying subresources for existing types. (e.g. /scale)
+- Material changes to how fields on existing types are interpreted / used - including, but not limited to
+  validation and defaulting of the fields.
+
+The following on their own do not qualify as API changes, but should follow the process defined by the related
+[special interest groups].
+
+- Adding or changing a command or flag to kubectl: See [contributing to SIG cli]
+- Building new command line tools: See [contributing to SIG cli]
+- Adding or changing a flag to kubelet: sig-node
+- Adding or changing a flag to apiserver: sig-api-machinery
+- Refactoring code
+- Building extensions
+
+## Life of an API change
+
+1. Start a discussion
+2. Write a short description
+3. Write a detailed design proposal in the [kubernetes/community] repo
+4. Schedule a design review
+5. File an issue in the [kubernetes/features] repo that links to the proposal
+6. Implement the proposal in the [kubernetes/kubernetes] repo
+7. Write user facing documentation on [kubernetes/kubernetes.github.io]
+
+### Start a discussion.
+
+Expected time for decision: 1-4 weeks (depending on the SIG and feature)
+
+Start a discussion with the relevant [special interest groups] that will [own] the changes.
+Depending on the SIG, this may be done via SIG meetings, email groups, slack channel, etc.
+
+### Write a short description
+
+Expected time for decision: 2 weeks
+
+Write a short description of the API changes by copying the [design summary template]
+into a new issue, and [@mention] @kubernetes/api-reviewers and the owning SIG reviewers
+group.
+
+Determine whether @kubernetes/api-reviewers is willing to accept a design proposal
+for your feature.  The answer could be "no, we don't want to solve this problem this way."
+or "no, this is something we want, but cannot commit the resources to reviewing it this release."
+
+### Write a detailed design proposal
+
+Expected time for initial feedback: 3-6 weeks
+
+Write a detailed design proposal PR using the [design proposal PR template].  [@mention]
+@kubernetes/api-reviewers and the appropriate SIGs on the PR and reference the original
+issue in the PR description.
+
+### Schedule a design review
+
+Expected time for next slot: 4-8 weeks
+
+Schedule a time for your design to be reviewed.
+
+Design approvers:
```

Comment:
We MUST link externally to a design approvers document for community transparency. This document should essentially state: "these are the current reviewers, we are still figuring out how to add/remove people to this list, see the [governance discussion](https://groups.google.com/forum/#!topic/kubernetes-dev/4e8WOnMvZC0)".

One of the most [confusing things about the API process](https://groups.google.com/d/msg/kubernetes-sig-api-machinery/sq9Jwgeyy7Q/wAbpho1dBwAJ) today is that we have inconsistent lists and no central doc that explains what the team is, etc.